src/extension/background-script/actions/nostr/signEventOrPrompt.ts

```diff
@@ -17,15 +23,60 @@ const signEventOrPrompt = async (message: MessageSignEvent) => {
     };
   }
 
+  const enableMessage: MessageAllowanceEnable = {
+    origin: message.origin,
+    args: {
+      host: message.origin.host,
+    },
+    action: "public/webln/enable",
+  };
+
+  await enable(enableMessage);
```

Comment:
can we move this into the content script? 
and call in [the nostr provider](https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/ln/nostr/index.ts#L6-L16) `enable()` before the actual call? 

then we would not need to do this "hack" to use the action in another action and such. 
The flow would then be the same as in webln.

Reply:
Done that, I hope, have a look please